pico_w/udp_beacon/picow_udp_beacon.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2022 Andrew McDonnell
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <string.h>
+#include <stdlib.h>
+
+#include "pico/stdlib.h"
+#include "pico/cyw43_arch.h"
+
+#include "lwip/pbuf.h"
+#include "lwip/udp.h"
+
+#define UDP_PORT 4444
+#define DEBUG_printf printf
```

Comment:
This `DEBUG_printf` define isn't actually used?

Reply:
OK, fixed